group_vars/checkmk/local_check_templates/sftp_with_pass_check.j2

```diff
@@ -0,0 +1,26 @@
+{{ ansible_managed | comment }}
+#!/bin/bash
+
+# Exit codes
+EXIT_OK=0
+EXIT_WARN=1
+EXIT_CRIT=2
+EXIT_UNKNOWN=3
+
+# Check if all credentials are defined
+if [[ -z "{{ sftp_host }}" || -z "{{ sftp_user }}" || -z "{{ sftp_password }}" ]]; then
```

Comment:
How will we pass in these ansible variables?

Reply:
🤦🏿‍♂️  I wrote the shell script before coffee and then yaml-ized it.

Reply:
I used perl. This is a pattern worth further discussion. The previous templates assume we would use what is on the OS but this may possibly not scale.

Reply:
will bring this up at Ansible Open House possibly